### PR TITLE
feat(ci/cd): Add container image packages to release github action

### LINKED ISSUE #8: Release config-file-validator as a container
## Description
Release the config-file-validator as a container to the [Github Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry). The tag should match the release version. We want to release three versions of the container per release:

1. A full-featured container based on ubuntu
2. A minimalistic container based on alpine
3. A scratch container with the statically-linked config-file-validator

The Github action for creating the containers should run on release.

### PR DESCRIPTION
**Pull request by gokulav137**
Resolves #8 

Images will be created for alpine, ubuntu and scratch and pushed to github registry

I tested the release out in my fork.

Packages: https://github.com/gokulav137?tab=packages&repo_name=config-file-validator

workflow run: https://github.com/gokulav137/config-file-validator/actions/runs/6431248766

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -34,4 +38,40 @@ jobs:
...
+            postfix: ""
+          - base: "scratch"
+            postfix: "-scratch"
+          - base: "ubuntu:20.04"
+            postfix: "-ubuntu"
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Log in to the Container registry
+      uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
```
Comment (jackswiney) [inline .github/workflows/release.yml:59]: What's the reason for using a SHA instead of a release version?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -34,4 +38,40 @@ jobs:
...
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
```
Comment (jackswiney) [inline .github/workflows/release.yml:66]: Same as above
> What's the reason for using a SHA instead of a release version?

**Review by kehoecj — APPROVED** [review]
LGTM - pulled tested the containers and everything worked as expected. Reminder to myself - after next release I need to update the https://github.com/Boeing/config-file-validator#container-run section to reference the newly released container

File: .github/workflows/release.yml (hunk 1 above)
Comment (gokulav137) [reply .github/workflows/release.yml:59]: This is as per the documentation in : https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages
```
# GitHub recommends pinning actions to a commit SHA.
# To get a newer version, you will need to update the SHA.
# You can also reference a tag or branch, but the action may change without warning.
```
I left the same SHA  as specified in the docs.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -34,4 +38,40 @@ jobs:
         ld_flags: -w -s -extldflags "-static"
         build_tags: -tags netgo
         project_path: cmd/validator
-        extra_files: LICENSE README.md
\ No newline at end of file
+        extra_files: LICENSE README.md
+
+  registery-matrix:
```
Comment (jd4235) [inline .github/workflows/release.yml:43]: typo
```suggestion
  registry-matrix:
```

**Comment by jd4235** [comment]
LGTM

File: .github/workflows/release.yml (hunk 3 above)
Comment (gokulav137) [reply .github/workflows/release.yml]: Thanks

**Pull request merged**